c_src/h3.c

```diff
@@ -617,6 +723,126 @@ erl_indices_are_neighbors(ErlNifEnv * env, int argc, const ERL_NIF_TERM argv[])
                                                                   : ATOM_FALSE;
 }
 
+static ERL_NIF_TERM
+erl_max_polyfill_size(ErlNifEnv * env, int argc, const ERL_NIF_TERM argv[])
+{
+    int        resolution;
+    GeoPolygon polygon;
+    int        result;
+    if (argc != 2 || !get_resolution(env, argv[1], &resolution))
+    {
+        return enif_make_badarg(env);
+    }
+    if (!get_geo_polygon(env, argv[0], &polygon))
+    {
+        return enif_make_badarg(env);
+    }
+    result = maxPolyfillSize(&polygon, resolution);
+    (void)free_geo_polygon(&polygon);
+    return enif_make_int(env, result);
+}
+
+static ERL_NIF_TERM
+erl_polyfill(ErlNifEnv * env, int argc, const ERL_NIF_TERM argv[])
+{
+    int          resolution;
+    GeoPolygon   polygon;
+    int          polyfillsize;
+    H3Index *    h3indices = NULL;
+    ERL_NIF_TERM list;
+    int          i;
+    if (argc != 2 || !get_resolution(env, argv[1], &resolution))
+    {
+        return enif_make_badarg(env);
+    }
+    if (!get_geo_polygon(env, argv[0], &polygon))
+    {
+        return enif_make_badarg(env);
+    }
+    polyfillsize = maxPolyfillSize(&polygon, resolution);
+    h3indices    = (void *)calloc(polyfillsize, sizeof(H3Index));
```

Comment:
I guess we should be consistent about calloc vs enif_alloc. Perhaps use enif_alloc consistently throughout?

Reply:
Yeah, I realized that `calloc` was being used elsewhere _after_ I had used `enif_alloc`.  I can change everything to `enif_alloc` if you'd like.

Reply:
That would be awesome. And ensure we handle NULL responses for the alloc calls.